packages/workbox-broadcast-cache-update/broadcastUpdate.mjs

```diff
@@ -55,6 +56,18 @@ import './_version.mjs';
  * @memberof workbox.broadcastUpdate
  */
 const broadcastUpdate = (channel, cacheName, url, source) => {
+  // There are browsers which support service workers but don't support the
+  // Broadcast Channel API.
+  // See https://github.com/GoogleChrome/workbox/issues/1304
+  if (!('BroadcastChannel' in self)) {
```

Comment:
You need to check that the channel exists along with this - since you are setting the channel to undefined if it's not supported.